<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SchemeObliqueWidgetCreateOrEditScheme</class>
 <widget class="QWidget" name="SchemeObliqueWidgetCreateOrEditScheme">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>250</width>
    <height>258</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Изменить всю схему</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="0">
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>Первый узел</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="1" column="0">
          <widget class="QRadioButton" name="nodeOf2_3">
           <property name="text">
            <string>Нити 2 - 3</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QRadioButton" name="nodeOf1_2">
           <property name="text">
            <string>Нити 1 - 2</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayoutCount">
        <item>
         <layout class="QVBoxLayout" name="verticalLayoutLabel">
          <item>
           <widget class="QLabel" name="labelCountThread">
            <property name="text">
             <string>Количество нитей</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labelCountRow">
            <property name="text">
             <string>Количество строк</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayoutBox">
          <item>
           <widget class="QSpinBox" name="countThreads">
            <property name="minimum">
             <number>3</number>
            </property>
            <property name="maximum">
             <number>2147483647</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="countHalfrow">
            <property name="minimum">
             <number>2</number>
            </property>
            <property name="maximum">
             <number>2147483647</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayoutName">
     <item>
      <widget class="QLabel" name="label_name">
       <property name="text">
        <string>Имя</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="name"/>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayoutButton">
     <item>
      <widget class="QPushButton" name="createOrEdit">
       <property name="text">
        <string>Создать</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancel">
       <property name="text">
        <string>Отмена</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
